open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/pom.xml

```diff
@@ -15,102 +15,106 @@
         <version>1.1-SNAPSHOT</version>
     </parent>
 
-    <properties>
-        <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
Is there a reason we have these settings here? They should already be inherited from the top level pom

Reply:
Removed :)